drivers/gpio/Kconfig.sim

```diff
@@ -0,0 +1,30 @@
+# Simulated GPIO configuration options
+
+# Copyright (c) 2020 Friedt Professional Engineering Services, Inc
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_SIM
+	bool "Simulated GPIO driver"
+	default n
+	help
+	  Enable simulated GPIO driver.
+
+if GPIO_SIM
+
+config GPIO_SIM_NPORTS
```

Comment:
Seems like both of these configs could be in the device tree?

Reply:
Interesting - what would be your desired property name in that case?

Reply:
Done. Used the common ngpios DT binding.